classes/app/exhibition/get/GetExhibition.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Exhibition\Get;
+
+use App\Exhibition\ExhibitionRepository;
+use App\Utils\InvalidArgumentException;
+
+class GetExhibition
+{
+    /**
+     * @var ExhibitionRepository
+     */
+    private $exhibitionRepository;
+
+    /**
+     * DeleteExhibition constructor.
+     * @param ExhibitionRepository $repository
+     */
+    public function __construct(ExhibitionRepository $repository)
+    {
+        $this->exhibitionRepository = $repository;
+    }
+
+    /**
+     * @param Request $request
+     * @return mixed
+     * @throws InvalidArgumentException
+     */
+    public function get(Request $request)
+    {
+        if (!isset($request->id)) {
+            throw new InvalidArgumentException("Exhibition id required");
+        }
+
+        return $this->exhibitionRepository->getExhibition($request->id);
```

Comment:
This is the point where we should wrap the data coming from the repository into an `Response`-Object which is the result of our use-case.

Think about a more complex use case which might access multiple repositories. The `Response`-Object should then aggregate the results into one response.